core/src/elements/ons-back-button.js

```diff
@@ -74,15 +74,17 @@ class BackButtonElement extends BaseElement {
    *  [ja]バックボタンの見た目を指定します。[/ja]
    */
 
-  createdCallback() {
-    contentReady(this, () => {
-      if (!this.hasAttribute('_compiled')) {
-        this._compile();
-      }
+  constructor(self) {
+    self = super(self);
+
+    contentReady(self, () => {
+      self._compile();
```

Comment:
It didn't work when I was creating a normal constructor. Probably because of this:

https://github.com/WebReflection/document-register-element#upgrading-the-constructor-context